scripts/apply_style.py

```diff
@@ -89,13 +91,17 @@ def find_clang_format():
             'cmake_format',
             '--separate-ctrl-name-with-space',
             '--line-width', '100',
-            '--in-place'
+            '-'
         ],
         'filename_patterns': ('*.cmake', 'CMakeLists.txt')
     }
 }
 
+NUM_FILES_PROCESSEFD_SO_FAR = 0  # Global variable
```

Comment:
We have generally been using `SCREAMING_SNAKE_CASE` for contants. Consider refactoring to normal `snake_case` to make it more consistent